crates/compiler/codegen/tests/snapshots/mdtest_codegen_snapshots@arrays_in_cairo_m___array_access_with_variable_index.snap

```diff
@@ -33,38 +33,38 @@ variable_index_access_0:
   10: 4 2 0 3              // [fp + 3] = [fp + 2] + 0 (copy array pointer)
   11: 9 30 15 _            // [fp + 15] = 30
   12: 44 2 2 15            // [[fp + 2] + 2] = [fp + 15]
-  13: 5 -4 0 4             // [fp + 4] = [fp + -4] op 0
-  14: 14 4 3 _             // if [fp + 4] != 0, jump to not_zero_0
+  13: 4 -4 0 4             // [fp + 4] = [fp + -4] + (-0) (-0 as M31 -> 0)
```

Comment:
why the there this -0 in the comment but not in the array?

Reply:
I'm not sure I understand, all instruction operands are encoded as M31 values